src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -298,6 +299,20 @@ const ProjectsComponent = ({
           </ListItem>
         }
 
+        { team.smooch_bot &&
```

Comment:
Request: Remove this condition please. We don't need to have the condition here because workspaces can have published reports by just using the API (as the ticket specified: "Always display this list").

Reply:
ok